core/ingest/src/main/java/mil/nga/giat/geowave/core/ingest/local/AbstractLocalFileDriver.java

```diff
@@ -163,6 +169,8 @@ protected void processInput(
 				return;
 			}
 
+			// HP Fortify "Null Dereference" false positive
+			// Null checking is done in the next line
 			String s3EndpointUrl = configProperties.getProperty(ConfigAWSCommand.AWS_S3_ENDPOINT_URL);
```

Comment:
its talking about configProperties, this one's a true-positive (line 151 configProperties is set to null and is only set to non-null within a conditional)